templates/vanilla/packages/client/src/mud/setupNetwork.ts

```diff
@@ -1,18 +1,52 @@
+/* Definitions required to connect to a blockchain */
+
+/*
+ * The MUD client code is built on top of Viem
+ * (https://viem.sh/docs/getting-started.html).
+ * This line imports the functions we need from it.
+ */
 import { createPublicClient, fallback, webSocket, http, createWalletClient, Hex, parseEther, ClientConfig } from "viem";
 import { createFaucetService } from "@latticexyz/services/faucet";
 import { encodeEntity, syncToRecs } from "@latticexyz/store-sync/recs";
+
+/* Get the network configuration. */
 import { getNetworkConfig } from "./getNetworkConfig";
 import { world } from "./world";
 import IWorldAbi from "contracts/abi/IWorld.sol/IWorld.abi.json";
 import { createBurnerAccount, createContract, transportObserver, ContractWrite } from "@latticexyz/common";
+
+/*
+ * Use the RxJS library (https://rxjs.dev/) to create
+ * event handlers.
+ */
 import { Subject, share } from "rxjs";
+
+/*
+ * Import packages/contracts/mud.config.ts with the World
+ * information. See
+ * https://mud.dev/tutorials/walkthrough/minimal-onchain#mudconfigts
+ */
```

Comment:
If we're gonna explain this, I would prefer to explain what this is for rather than explaining the literal import

> Import our MUD config, which includes strong types for our tables and other config options. We use this to generate things like RECS components and get back strong types for them.